packages/cli/src/commands/pull.ts

```diff
@@ -30,7 +37,8 @@ const help = () => {
     'DIR'
   )}    Path to the global ${'`.vercel`'} directory
     -d, --debug                    Debug mode [off]
-    --env [filename]               The file to write Development Environment Variables to [.env]
+    --env-file [filename]               The file to write Development Environment Variables to [.env]
+    --environment ${getEnvTargetPlaceholder()}
```

Comment:
Do we have a default if `--environment` isn't provided? It should be listed in brackets

Reply:
We do! Should the options use brackets too? This is what the options section looks like after adding the default value:

```
-h, --help                     Output usage information
-A FILE, --local-config=FILE   Path to the local `vercel.json` file
-Q DIR, --global-config=DIR    Path to the global `.vercel` directory
-d, --debug                    Debug mode [off]
--env-file [filename]          The file to write Development Environment Variables to [.env]
--environment <production | preview | development> Deployment environment [development]
-y, --yes                      Skip the confirmation prompt
```

The result of `getEnvTargetPlaceholder` is `<production | preview | development>`, but should it be something like `[production | preview | development]` instead?

Reply:
Looks like `vc env --help` doesn't list the possible enum options in the initial commands at the top, but it does list them in the examples at the bottom, such as

```
  – Add a new variable for a specific Environment

      $ vercel env add <name> <production | preview | development>
      $ vercel env add DB_PASS production
```